pkgs/games/openra/engines/release/default.nix

```diff
@@ -0,0 +1,14 @@
+{ fetchFromGitHub }:
+
+rec {
+  name = "release";
+  version = "20230225";
```

Comment:
```suggestion
  pname = "release";
  version = "unstable-2023-02-25";
```

Reply:
The `pname` is always `openra`, `name` here is the name of the OpenRA release type. I've changed how the `buildOpenRAEngine` function is called to avoid the confusion.